pkg/webhook/controlplane/ensurer.go

```diff
@@ -69,6 +69,29 @@ func (e *ensurer) EnsureKubeControllerManagerDeployment(ctx context.Context, ect
 	return nil
 }
 
+// // EnsureKubeAPIServerService ensures that the kube-apiserver service conforms to the provider requirements.
```

Comment:
Why is the following code fragment commented?

Reply:
Oops, I forgot to remove it. 
Thanks!

Reply:
Fixed.